docker/entry.sh

```diff
@@ -4,7 +4,7 @@ sa_dir=/var/run/secrets/kubernetes.io/serviceaccount
 
 kc_args="--server=https://kubernetes.default.svc.cluster.local --token=$(cat $sa_dir/token) --certificate-authority=$sa_dir/ca.crt"
 
-kubectl $kc_args create -f manifests/
+kubectl $kc_args apply -f manifests/
```

Comment:
Won't it throw an error/warning saying use `create` in case the resources don't already exist? Should we also add `|| true` ?

Reply:
@umangachapagain The purpose we used to apply is when the container gets rebuild it should apply the new changes. Where create says its already there. So will not be able to update. 

I will add the` || true`

Reply:
Isn't the `|| true` is going to mask the failure, causing the following `$?` test to always pass?

`apply` should act just like `create` for resources that aren't there... Am I misunderstanding something?